src/wazuh_modules/wm_sca.c

```diff
@@ -844,13 +845,13 @@ static int wm_configuration_assessment_do_scan(OSList *p_list,cJSON *profile_che
                     char *f_value = NULL;
                     char *dir = NULL;
 
-                    file = wm_configuration_assessment_get_pattern(value);
+                    file = wm_sca_get_pattern(value);
                     if (!file) {
                         merror(WM_CONFIGURATION_ASSESSMENT_MONITORING_INVALID_RKCL_VAR, value);
```

Comment:
`WM_CONFIGURATION_ASSESSMENT_MONITORING_INVALID_RKCL_VAR` -> `WM_SCA_INVALID_VAR` for example